orderer/common/multichannel/registrar_test.go

```diff
@@ -1543,6 +1571,119 @@ func TestRegistrar_RemoveChannel(t *testing.T) {
 		require.Nil(t, registrar.GetChain("channel-im-not-a-member-of"))
 		require.NotContains(t, ledgerFactory.ChannelIDs(), "channel-im-not-a-member-of")
 	})
+
+	t.Run("remove system channel ledger fails", func(t *testing.T) {
+		setup(t)
+		defer cleanup()
+
+		newLedger(ledgerFactory, "raft-sys-channel", genesisBlockSysRaft)
+		registrar := NewRegistrar(config, ledgerFactory, mockCrypto(), &disabled.Provider{}, cryptoProvider, dialer)
+		registrar.Initialize(mockConsenters)
+		require.NotNil(t, registrar.GetChain("raft-sys-channel"))
+		require.Contains(t, ledgerFactory.ChannelIDs(), "raft-sys-channel")
+
+		os.RemoveAll(filepath.Join(tmpdir, "pendingops", "remove"))
+
+		err := registrar.RemoveChannel("raft-sys-channel")
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "error removing ledger for system channel raft-sys-channel")
+
+		// After removing the system channel, it no longer exists in the registrar or the ledger
```

Comment:
It seems this is the wrong comment for what's going on here

Reply:
🧹  removed.